### PR TITLE
Pre-release modifications

### PR DESCRIPTION
**Pull request by cardosolucas**
Some modifications to create engine-executor docker image and package name changing.

### CODE REVIEW

**Review by Wei-1 — COMMENTED** [review]
There is something weird going on with the `virtualenv` version on `travis-ci` that is causing this.
The only difference that I can find is that `virtualenv` got updated from `20.8.1` to `20.9.0` or `20.10.0` and things stop working.
Let's test around to see if we can solve this.

**Comment by Wei-1** [comment]
I tested with a minimum tweak, but it doesn't work in #60
I had also print more logs from `tox` in that PR, you can check if there is anything helpful, although it doesn't seem useful to me.

**Comment by cardosolucas** [comment]
It seems like the problem is with the installation of cryptography library. The openssl required version is > 1.1.0 but we have > 1.0.0. I have tried some fixes but it doesn't work. I'll continue to work on it tomorrow.

File: .travis.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ matrix:
     - language: python
       os: linux
       python:
-        - 3.6
+        - "3.8"
```
Comment (Wei-1) [inline .travis.yml]: Weird, the build configuration had not been changed: https://github.com/apache/incubator-marvin/runs/4835434559

Comment (cardosolucas) [reply .travis.yml]: It was getting stuck on validation.

**Comment by cardosolucas** [comment]
It's working now. I changed the ubuntu version to get the right openssl version installed.

**Comment by Wei-1** [comment]
I will check and merge it before weekend!

**Pull request merged**